auto_nag/scripts/spambug.py

```diff
@@ -0,0 +1,89 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag import people
+from auto_nag.bugbug_utils import get_bug_ids_classification
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.utils import nice_round
+
+
+class SpamBug(BzCleaner):
+    def __init__(self):
+        super().__init__()
+        self.people = people.People.get_instance()
+
+    def description(self):
+        return "[Using ML] Detect spam bugs"
+
+    def columns(self):
+        return ["id", "summary", "confidence", "autofixed"]
+
+    def sort_columns(self):
+        return lambda p: (-p[2], -int(p[0]))
+
+    def handle_bug(self, bug, data):
+        reporter = bug["creator"]
```

Comment:
You're using `creator`, but `creator` is not in the include_fields in the Bugzilla query, so it won't be available. You'll need to add it in the include_fields list.